src/neo/Ledger/Blockchain.cs

```diff
@@ -370,14 +446,19 @@ private void Persist(Block block)
                 Debug.Assert(header.Index == block.Index);
         }
 
+        /// <summary>
+        /// Gets a <see cref="Akka.Actor.Props"/> object used in creating the <see cref="Blockchain"/> actor.
```

Comment:
I did not understand very well the  " used in creating"

Reply:
Fixed.